src/components/global/Navbar.jsx

```diff
@@ -1,8 +1,157 @@
-const Navbar = () => {
-  // any logic needed
-  // return (
-  // all the JSX (looks like HTML) to render
-  // )
+import {
+  AppBar,
+  Box,
+  Hidden,
+  IconButton,
+  List,
+  ListItem,
+  Popover,
+  Toolbar,
+  Typography
+} from '@mui/material';
+import MenuIcon from '@mui/icons-material/Menu';
+import Brightness3Icon from '@mui/icons-material/Brightness3';
+import Brightness7Icon from '@mui/icons-material/Brightness7';
+import { Link } from 'react-router-dom';
+import { useState, useRef } from 'react';
+
+const logoBlobStyle = {
+  position: 'absolute',
+  bottom: 0,
+  left: -30,
+  right: 0,
+  top: -50,
+  height: '250px',
+  backgroundImage: 'url(/assets/logoBlobNav.svg)',
+  backgroundRepeat: 'no-repeat',
+  maxWidth: '200px'
+};
+
+const navTextStyle = {
+  textDecoration: 'none',
+  color: 'inherit'
 };
 
-export default Navbar;
+const menuTextStyle = {
+  textDecoration: 'none',
+  color: 'inherit',
+  justifyContent: 'center',
+  pb: '10px'
+};
+function NavBar() {
+  const [darkMode, setDarkMode] = useState(false);
+  const [menuOpen, setMenuOpen] = useState(false);
+
+  // anchor for menu to pop from its icon
+  const anchorRef = useRef(null);
+
+  const handleThemeChange = () => {
+    setDarkMode(!darkMode);
+  };
+
+  const handleToggle = () => {
+    setMenuOpen(!menuOpen);
+  };
+
+  const handleClose = () => {
+    setMenuOpen(false);
+  };
+
+  const menuItems = (
+    <List>
+      <ListItem sx={menuTextStyle} component={Link} to="/" onClick={handleClose}>
```

Comment:
Maybe some styling on hover?   Either change link color, or maybe even the background color of that link section in menu?

Actually probably do this for all the links in the navbar, not just the mobile menu (like we have in footer.). Better indicator than solely the pointer cursor.

Reply:
added